tests/unit/telegram_bot/test_message_sender.py

```
@@ -0,0 +1,154 @@
+        ) as MockImage, patch(
+            "core.services.message_sender.image_compressing",
+            return_value=mock_image,
+        ), patch(
+            "core.services.message_sender.settings"
+        ) as mock_settings:
+            MockImage.open.return_value = mock_image
+            mock_settings.TELEBOT_API_SERVER = None
+            mock_settings.TELEGRAM_IMAGE_DIMENSION_LIMIT = 2000
+            mock_settings.TELEGRAM_IMAGE_SIZE_LIMIT = 10 * 1024 * 1024  # 10MB
+
+            media_group, file_group = await media_files_packaging(media_files, data)
```

Ruff (0.15.7)

[warning] Unpacked variable `media_group` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)